appuio/haproxy/values.yaml

```diff
@@ -85,6 +85,8 @@ haproxy:
     nodeCount: 3
     metrics:
       enabled: false
+      exposeLoadbalancer: false
```

Comment:
Since this used to be enabled by default, when enabling metrics, I would have the default set to true to make this a non-breaking change.